src/devices/OneWire/FamilyBindings/OneWireThermometerDevice.Linux.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Iot.Device.OneWire
+{
+    public partial class OneWireThermometerDevice : OneWireDevice
```

Comment:
I'lll take a deeper look later but for a quick feedback, I think this device should go by name (i.e. DS18B20) and be under Iot.Device.Ds18B20. The 1-wire should be a separate directory which this project would reference.

Reply:
Actually second look - not seeing much device specific. Please change only if this is DS18B20 specific

Reply:
(please at least mention DS18B20 somewhere in the readme since it will likely cause that someone will duplicate the effort)

Reply:
Yes, I actually wrote this for the MAX31820. The binding supports 5 1-wire device families (temp sensors). MAX31820 and DS18B20 are both of the same family. So I can't use the device name in the binding. But can definitely add some device names in the readme.
My intent was not to hijack ericstj's issue #197. I had to implement this anyways and just uploaded it as a WIP PR in case eric could make use of it. I can continue the work, but then I want eric's blessing ;). Now it has all I need for my project.

Reply:
I've just talked with @ericstj and he is fine with you picking this up 😄